java/compaction/compaction-job-creation/src/test/java/sleeper/compaction/strategy/impl/SizeRatioCompactionStrategyTest.java

```diff
@@ -45,11 +46,16 @@
 
 public class SizeRatioCompactionStrategyTest {
 
+    private static final Schema DEFAULT_SCHEMA = schemaWithKey("key");
     private final InstanceProperties instanceProperties = createTestInstanceProperties();
     private final TableProperties tableProperties = createTestTableProperties(instanceProperties, schemaWithKey("key"));
```

Comment:
This is defining the schema twice. It's a bit odd that in BasicCompactionStrategyTest it's using table properties without any schema, but here it is setting one.